app/src/main/java/org/medfordhistorical/overthemystic/MainActivity.java

```diff
@@ -0,0 +1,36 @@
+package org.medfordhistorical.overthemystic;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+
+import static android.provider.AlarmClock.EXTRA_MESSAGE;
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+    }
+
+    /** Called when the user taps one of the home screen buttons */
+    public void launchTour(View view) {
```

Comment:
rename to something like launchTourSelection since we're not quite launching the tour yet until they've picked